manifests/process.pp

```diff
@@ -0,0 +1,71 @@
+# @summary Creates an system (OS level) user for use with StackStorm
```

Comment:
Looks like this comment is copy./pasted and is from user.

Maybe better name would be `service` instead of `process`?

Reply:
I went back and forth on this but chose process because that is what ST2 calls it in their documentation: https://docs.stackstorm.com/reference/ha.html#st2rulesengine

Willing to change if you want?

Reply:
@bishopbm1 i'd call it `service` because that's what Puppet calls it